### ISSUE TITLE
PyInit function generation drops first and last letter from module name 

### ISSUE THREAD

**Issue by zboody** [PyO3/pyo3#163]
It looks like the first and last letters of the argument given to `pyo3::py::modinit` get dropped when generating the PyInit_ function.

### 🌍 Environment

 - Lubuntu 17.10
 - Python 3.6.3 from apt
 - rustc 1.27.0-nightly (ac5c0848d 2018-05-14)
 - pyo3 0.2.6 (Should be the latest on crates.io)
 
### 💥 Reproducing 

lib.rs
```
#![feature(proc_macro)]

extern crate pyo3;
use pyo3::{PyResult, Python, PyModule};

use pyo3::py::modinit as pymodinit;

#[pymodinit(clippingrepro)]
fn init_mod(py: Python, m: &PyModule) -> PyResult<()> {

    #[pyfn(m, "laden_swallow")]
    fn laden_swallow_py(_py: Python) -> PyResult<String> {
        Ok(String::from("African or european?"))
    }

    Ok(())
}
```

Cargo.toml
```
[package]
name = "clippingrepro"
version = "0.1.0"
authors = ["zboody"]

[lib]
name = "clippingrepro"
crate-type = ["cdylib"]

[dependencies.pyo3]
version = "0.2"
features = ["extension-module"]
```

Building that and trying to import the module throws the error:
`ImportError: dynamic module does not define module export function (PyInit_clippingrepro)`

And sure enough, looking for the init function with bingrep gives you: 
```
                                0x0                               0x0
              58b0    GLOBAL     FUNC        PyInit_lippingrepr
```

This happens in both debug and release builds. (Renaming the .so to `lippingrepr.so` and importing it works, though.)

**Comment by althonos** [comment]
Hi @zboody , 
this is a known bug and will be fixed by #160.

In the meantime, you can either:
* wrap your module name in quotes (won't be forward compatible)
* use the `syn13` branch of the git repository.

**Comment by konstin** [comment]
I've released v0.2.7 with the fix.
